### ISSUE TITLE
Returning all possible lemmas for a single word

### ISSUE THREAD

**Issue by zeeyado** [adbar/simplemma#132]
I've had a look at simplemma previously, and I'm sure there was a method to load a language (simplemma.load_data or similar), then a method for returning all possible lemmas (simplemma.lemmatize_all I think). But it has been a while.

Is it possible to achieve this?

To be clear, I mean that a single word search like "ground" should return "grind" and "ground" as lemmas, and something like "saw" should return "saw" and "see", and so on.

**Comment by juanjoDiaz** [comment]
I don't think that this is possible as things stand because simplemma's datasets are designed to only have one lemma per word.

**Comment by adbar** [comment]
Yes, it's indeed not possible in the current state of the package.
